lib/plugins/aws/custom-resources/index.js

```diff
@@ -28,8 +28,11 @@ async function addCustomResourceToService(
 
   const { serverless } = awsProvider
   const providerConfig = serverless.service.provider
+
+  // write custom resource lambda logs by default
   const shouldWriteLogs =
-    providerConfig.logs && providerConfig.logs.frameworkLambda
+    (providerConfig.logs && providerConfig.logs.frameworkLambda) ?? true
```

Comment:
@Mmarzex This one line change will give the custom resource lambda permission to write logs to cloudwatch by default, making it easier for us and our users to debug issues in the custom resource lambda function. This makes a lot of sense to me.

Could you think of any reason why we haven't been doing that the whole time? Any side effects you could think of?